app/models/medium.rb

```diff
@@ -282,7 +282,16 @@ def self.lecture_search_option
   # returns search results for the media search with search_params provided
   # by the controller
   def self.search_by(search_params, page)
-    search_params[:types] = [] if search_params[:all_types] == '1'
+    # If the search is initiated from the start page, you can only get
+    # generic media sorts as results even if the 'all' radio button
```

Comment:
"types" instead of "sorts" (to avoid confusion with "ordering")

EDIT: Oh, I've seen "sort" is used a lot in the code, might be better to leave it like this then to stay consistent.